packages/admin/admin/src/FinalForm.tsx

```diff
@@ -48,6 +49,7 @@ export function FinalForm<FormValues = AnyObject>(props: IProps<FormValues>) {
             editDialog?.closeDialog({ delay: true });
         },
         validateWarning,
+        allowPristineSubmission = false,
```

Comment:
What's your motivation for this change?

Reply:
The motivation for this change doesn't exist anymore. Therefore I removed this change